java/build.gradle

```
@@ -2,8 +2,59 @@ plugins {
     id 'java'
     id 'java-library'
     id 'io.freefair.lombok' version '8.6'
-    id 'jacoco'
     id 'com.diffplug.spotless' version '6.25.0'
+    id 'jacoco'
+}
+
+allprojects {
+    // JaCoCo section (common config for code coverage)
+    apply plugin: 'jacoco'
+
+    jacoco {
+        toolVersion = "0.8.9"
+    }
+
+    jacocoTestReport {
+        reports {
+            xml.configure { enabled false }
+            csv.configure { enabled false }
+            html.configure { enabled true } 
+        }
+
+        afterEvaluate {
+            classDirectories.setFrom(files(classDirectories.files.collect {
+                fileTree(dir: it)
+            }))
+        }
+    }
+
+    jacocoTestCoverageVerification {
+        violationRules {
+            rule {
+                element = 'CLASS'
+                excludes = [
+                    'glide.models.protobuf.*',
+                    'io.netty.*'
```

I meant exclude classes which work with netty
```suggestion
                    'glide.connectors.resources.*'
                    'glide.connectors.handlers.*'
```